src/Model/User.hs

```diff
@@ -733,3 +738,85 @@ pledgeStatus uid = do
     return $ case underfunded of
         [] -> AllFunded
         _ -> ExistsUnderfunded
+
+
+claimedTickets :: UserId -> Handler [(Entity Ticket, Maybe (Entity WikiTarget), Import.Value Text)]
+claimedTickets user_id = do
+
+    -- TODO: abstract out grabbing the project
```

Comment:
What is this comment referring to? I know it's copied over, but is the abstraction you are currently doing the abstraction it is talking about? If so, delete the comment. If not.... maybe delete the comment anyway.